src/main/webapp/app/css/calendar.css

```diff
@@ -0,0 +1,158 @@
+body {
+    background: #e0e0e0
+}
+
+#cal {
+    -moz-box-shadow: 0 3px 3px rgba(0, 0, 0, .25);
+    -webkit-box-shadow: 0 3px 3px rgba(0, 0, 0, .25);
+    margin: 50px auto;
+    font: 13px/1.5 helvetica neue, Helvatica, Arial, san-serif;
+    display: table;
+    position: absolute
+}
+
+#cal .header {
+    cursor: default;
+    background: #cd310d;
+    background: -moz-linear-gradient(top, #b32b0c, #cd310d);
+    background: -webkit-gradient(linear, left top, left bottom, from(#b32b0c), to(#cd310d));
+    height: 34px;
+    position: relative;
+    color: #fff;
+    -webkit-border-top-left-radius: 5px;
+    -webkit-border-top-right-radius: 5px;
+    -moz-border-radius-topleft: 5px;
+    -moz-border-radius-topright: 5px;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    font-weight: 700;
+    text-shadow: 0 -1px 0 #87260c;
+    text-transform: uppercase
```

Comment:
Please sort CSS properties alphabetically, so that they can be easily found later.